ale/base/data_naif.py

```diff
@@ -26,6 +26,25 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def metakernel(self):
         pass
 
+    @property
+    def light_time_correction(self):
+        """
+        Returns the type of light time correciton and abberation correction to
+        use in NAIF calls.
+
+        This defaults to light time correction and abberation correction (LT+S),
+        concrete drivers should override this if they need to either not use
+        light time correction or use a different type of light time correction.
```

Comment:
This is a change from ISIS. Right now, ISIS determines if a sensor uses light time or not based on a [keyword](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/base/objs/Spice/LightTimeCorrectionState.cpp#L125) in the IAK. We can continue to do that and not have individual drivers override this.

The main problem with continuing what ISIS does is that it's purely an on or off option for the entire sensor. By putting this into the driver, we can make light time correction choices on a per observation basis.